content/en/docs/how-tos/staged-update.md

```diff
@@ -175,14 +179,25 @@ spec:
   placementName: example-placement
   resourceSnapshotIndex: "1"
   stagedRolloutStrategyName: example-strategy
+  state: Initialize  # Initialize but don't start execution yet
 EOF
 ```
 
+The UpdateRun starts in `Initialize` state, which computes the stages without executing. This allows you to review the computed stages before starting:
+```bash
+kubectl get csur example-run -o yaml  # Review computed stages in status
```

Comment:
Also show an example with `-o yaml`? I assume the updateRun is only initialized but not progressing?